src/styles/blocks/animation.css

```diff
@@ -0,0 +1,10 @@
+@-webkit-keyframes time {
+  to {
+    transform: rotate(350deg);
+  }
+}
+@keyframes time {
+  to {
+    transform: rotate(350deg);
+  }
+}
```

Comment:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file